stackstate.conf.example

```diff
@@ -142,25 +137,23 @@ gce_updated_hostname: yes
 # procfs_path: /proc
 
 # ========================================================================== #
-# DogStatsd configuration                                                    #
-# DogStatsd is a small server that aggregates your custom app metrics. For   #
-# usage information, check out http://docs.datadoghq.com/guides/dogstatsd/   #
+# StsStatsd configuration                                                    #
+# StsStatsd is a small server that aggregates your custom app metrics. For   #
 # ========================================================================== #
 
-# If you don't want to enable the DogStatsd server, set this option to no
+# If you don't want to enable the StsStatsd server, set this option to no
 # use_dogstatsd: yes
```

Comment:
let's discuss with @markbakker78 if we really want to rename dogstatsd and how far we want to take this